index.html

```diff
@@ -1,91 +1,296 @@
 <!DOCTYPE html>
 <html lang="en">
+
```

Comment:
- This is not OPTIONAL nor a request change, just to **improve** your PR professionalism I suggest committing often in every step change you made in your code, and kindly choose a descriptive commit message.